packages/rfsm/rfsm.1.6.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "A toolset for describing and simulating StateChart-like state diagrams"
+description:
+  "A toolset for describing and simulating StateChart-like state diagrams"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/rfsm"
+doc: "http://jserot.github.io/rfsm"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "1.11"}
+  "menhir" {>= "2.0"}
+  "lascar" {>= "0.6"}
```

Comment:
```suggestion
  "lascar" {>= "0.6.0"}
```
Shouldn't it be excluding `0.6-alpha`?

Reply:
Yes. Sorry, i misinterpreted the version numbering scheme.